src/rtcModule/webrtc.cpp

```diff
@@ -1794,7 +1892,7 @@ void Call::removeSession(Session& sess, TermCode reason)
     }
 
     TermCode terminationCode = (TermCode)(reason & ~TermCode::kPeer);
-    if (terminationCode == TermCode::kErrIceFail || terminationCode == TermCode::kErrIceTimeout)
+    if (terminationCode == TermCode::kErrIceFail || terminationCode == TermCode::kErrIceTimeout || terminationCode == kErrSessSetupTimeout)
```

Comment:
Is it related to this PR?

Reply:
It isn't related to reduce latency but I paid attention during development. If you think is better remove to other PR I remove the line.
I added that condition because when there are invalid Turn server the session terminate with that termCode, then it's necessary to increase mIceFails and use static turn server the next time session start up